### ISSUE TITLE
Allow "note" variables with blank "name"

### ISSUE THREAD

**Issue by internetlindsey** [matthew-white/odkmeta#30]
xlsform allows a blank "name" for "note" type. odkmeta returns the below error in Stata when it encounters such cases: 

invalid name attribute ''
invalid name() suboption
invalid survey() option
r(198);

**Comment by jquattro** [comment]
Is there a workaround for this? I went into the survey.csv sheet and created names for the notes. But then when I run the do file that odkmeta creates, I get an error message "variable [name I created for the first note] not found." And none of my variables have labels.

**Comment by matthew-white** [comment]
@jquattro giving these notes names will result in an error, because the `odkmeta` do-file will try to attach metadata to these nonexistent variables. How about deleting these rows entirely from the survey .csv file?

**Comment by jquattro** [comment]
Yes! Deleting the rows worked. On another note: beware 'select multiple'
choices that include negative numbers (e.g. we used -66 for "don't know").
Excel and OpenOffice convert those to dates, which leads to the odkmeta
error "may not convert string."
Thank you for this wonderful time-saving program.

2015-10-19 22:26 GMT-04:00 Matthew White <notifications@github.com>:
